pyccel/codegen/printing/fcode.py

```diff
@@ -74,112 +77,112 @@
     # ---
-    'NumpyExp' : 'exp',
-    'NumpyLog' : 'Log',
+    'NumpyExp': 'exp',
+    'NumpyLog': 'Log',
     # 'NumpySqrt': 'Sqrt',  # sqrt is printed using _Print_NumpySqrt
     # ---
-    'NumpySin'    : 'sin',
-    'NumpyCos'    : 'cos',
-    'NumpyTan'    : 'tan',
-    'NumpyArcsin' : 'asin',
-    'NumpyArccos' : 'acos',
-    'NumpyArctan' : 'atan',
+    'NumpySin': 'sin',
+    'NumpyCos': 'cos',
+    'NumpyTan': 'tan',
+    'NumpyArcsin': 'asin',
+    'NumpyArccos': 'acos',
+    'NumpyArctan': 'atan',
     'NumpyArctan2': 'atan2',
-    'NumpySinh'   : 'sinh',
-    'NumpyCosh'   : 'cosh',
-    'NumpyTanh'   : 'tanh',
+    'NumpySinh': 'sinh',
+    'NumpyCosh': 'cosh',
+    'NumpyTanh': 'tanh',
     'NumpyArcsinh': 'asinh',
     'NumpyArccosh': 'acosh',
     'NumpyArctanh': 'atanh',
 }
 
 math_function_to_fortran = {
-    'MathAcos'   : 'acos',
-    'MathAcosh'  : 'acosh',
-    'MathAsin'   : 'asin',
-    'MathAsinh'  : 'asinh',
-    'MathAtan'   : 'atan',
-    'MathAtan2'  : 'atan2',
-    'MathAtanh'  : 'atanh',
+    'MathAcos': 'acos',
+    'MathAcosh': 'acosh',
+    'MathAsin': 'asin',
+    'MathAsinh': 'asinh',
+    'MathAtan': 'atan',
+    'MathAtan2': 'atan2',
+    'MathAtanh': 'atanh',
     'MathCopysign': 'sign',
-    'MathCos'    : 'cos',
-    'MathCosh'   : 'cosh',
-    'MathErf'    : 'erf',
-    'MathErfc'   : 'erfc',
-    'MathExp'    : 'exp',
+    'MathCos': 'cos',
+    'MathCosh': 'cosh',
+    'MathErf': 'erf',
+    'MathErfc': 'erfc',
+    'MathExp': 'exp',
     # 'MathExpm1'  : '???', # TODO
-    'MathFabs'   : 'abs',
+    'MathFabs': 'abs',
     # 'MathFmod'   : '???',  # TODO
     # 'MathFsum'   : '???',  # TODO
-    'MathGamma'  : 'gamma',
-    'MathHypot'  : 'hypot',
+    'MathGamma': 'gamma',
+    'MathHypot': 'hypot',
     # 'MathLdexp'  : '???',  # TODO
-    'MathLgamma' : 'log_gamma',
-    'MathLog'    : 'log',
-    'MathLog10'  : 'log10',
+    'MathLgamma': 'log_gamma',
+    'MathLog': 'log',
+    'MathLog10': 'log10',
     # 'MathLog1p'  : '???', # TODO
     # 'MathLog2'   : '???', # TODO
     # 'MathPow'    : '???', # TODO
-    'MathSin'    : 'sin',
-    'MathSinh'   : 'sinh',
-    'MathSqrt'   : 'sqrt',
-    'MathTan'    : 'tan',
-    'MathTanh'   : 'tanh',
+    'MathSin': 'sin',
+    'MathSinh': 'sinh',
+    'MathSqrt': 'sqrt',
+    'MathTan': 'tan',
+    'MathTanh': 'tanh',
     # ---
-    'MathFloor'    : 'floor',
+    'MathFloor': 'floor',
     # ---
     # 'MathIsclose' : '???', # TODO
     # 'MathIsfinite': '???', # TODO
     # 'MathIsinf'   : '???', # TODO
     # --------------------------- internal functions --------------------------
-    'MathFactorial' : 'pyc_factorial',
-    'MathGcd'       : 'pyc_gcd',
-    'MathDegrees'   : 'pyc_degrees',
-    'MathRadians'   : 'pyc_radians',
-    'MathLcm'       : 'pyc_lcm',
+    'MathFactorial': 'pyc_factorial',
+    'MathGcd': 'pyc_gcd',
+    'MathDegrees': 'pyc_degrees',
+    'MathRadians': 'pyc_radians',
+    'MathLcm': 'pyc_lcm',
```

Comment:
I'm guessing your editor did this automatically. We line these up purposely as we find it makes the code more readable. Could you find a way to turn off these automatic modifications in your editor please.

Having lots of trivial changes (e,g, whitespace changes) also makes it much harder to review your PR. According to github, you have modified 415 lines, which means the PR is much bigger than it needs to be

Reply:
I apologize for this, i will fix it right away!

Reply:
Thanks 👍